src/App.jsx

```diff
@@ -47,7 +47,8 @@ function App() {
                     <SearchBar />
                   </Route>
                   <Route path="/Contact" />
-                  <Route exact path="/" />
+                  <Route exact path="/" component={PictureDay} />
+                  <Route />
```

Comment:
this empty route looks useless, isn't it?

Reply:
Line 49? component contact will come later

Reply:
line 51, but you're right: line 49 is also useless. if component will come later, then add the route later ;)

Reply:
ok, exact, line 51 is useless